searchfeature1.java

```diff
@@ -0,0 +1,6 @@
+public class SearchFeature1{
+    //code for search feature 
+    // Using a List
+    //Connecting to Db
```

Comment:
Please use the using block to help garbage collect the object. Also use Dictionary Collection as your primary collection in the class.

Reply:
Fixed the code a/c to your review comments. Please suggest any other change if required.